### PR TITLE
Support clamp

### PR DESCRIPTION
**Pull request by bertmaher**

### CODE REVIEW

File: torch_tvm/operators.cpp [hunk 1]
```diff
@@ -444,6 +444,16 @@ RegisterTVMOperator reg({
        auto out = tvm::relay::CallNode::make(op, inputs, tvm::Attrs(), {});
        return out;
      }},
+    {Symbol::fromQualString("aten::clamp"),
+     [](Node* node, tvm::Array<tvm::relay::Expr> inputs) {
+       auto op = tvm::relay::Op::Get("clip");
+       auto clip_attrs = tvm::make_node<tvm::relay::ClipAttrs>();
+       clip_attrs->a_min = relayToConstant<float>(inputs[1]);
```
Comment (yinghai) [inline torch_tvm/operators.cpp]: How do we know the second and third input is constant?

Comment (kimishpatel) [reply torch_tvm/operators.cpp]: At this point all scalars are make into `tvm::relay::ConstantNode` and I dont think they are part of `ArgumentSpec`, although I am not sure. So you can argue that two calls of `aten::clamp` can come with two different values of clamp and we certainly won't catch that.
But in the JIT graph you can check if those inputs come from `prim::Constant` nodes or not. That would be a good thing to add.

Comment (yinghai) [reply torch_tvm/operators.cpp]: @kimishpatel It makes more sense to me. Do we actually do this for other ops?

Comment (kimishpatel) [reply torch_tvm/operators.cpp]: we do `relayToConstant` everywhere, so it would be good to add this test. I will put it up for TODO.

Comment (bertmaher) [reply torch_tvm/operators.cpp]: Should this diff depend on adding those `prim::Constant` checks, then?

Comment (kimishpatel) [reply torch_tvm/operators.cpp]: I dont think so.

**Comment by bertmaher** [comment]
Thanks for the review!  Can you merge (or give me write access so I can)? :)

**Comment by kimishpatel** [comment]
Not sure how to give you the write access (I woke up one day and I had the write access :)), but I will merge.

**Pull request merged**